### PR TITLE
added the assignment

### PR DESCRIPTION
**Pull request by Jaikishan0511**

### CODE REVIEW

File: js-promises/files.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+//check if file exists
+if (fs.existsSync(myFile)) {
+  console.log("Found file");
+}
+
+//check if file has permissions to read
+fs.access(myFile, fs.constants.R_OK, err => {
+  if (err) {
+    console.log("fie doesn't exist");
```
Comment (Shrikant9) [inline js-promises/files.js]: Print error message here instead.
``console.log(`file read problem: ${err.message}`);``

File: js-promises/files.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  } else {
+    console.log("can read file");
+  }
+});
+
+//if permission is there console log the content on terminal
+fs.readFile(myFile, "utf8", function(err, myFile) {
+  if (err) return console.error(err);
```
Comment (Shrikant9) [inline js-promises/files.js]: Do the same here.

**Review by Shrikant9 — CHANGES_REQUESTED** [review]

File: js-promises/files.js (hunk 1 above)
Comment (Jaikishan0511) [reply js-promises/files.js]: done

File: js-promises/files.js (hunk 2 above)
Comment (Jaikishan0511) [reply js-promises/files.js]: done

**Review by Shrikant9 — APPROVED** [review]

**Pull request merged**